src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioAnalyzer.cs

```diff
@@ -104,63 +97,5 @@ private void ResetReferenceAndErrors(out AnalyzerReference? reference, out Immut
                 _analyzerReference = null;
             }
         }
-
-        /// <summary>
-        /// This custom loader just wraps an existing loader, but ensures that we start listening to the file
-        /// for changes once we've actually looked at the file.
-        /// </summary>
-        private class AnalyzerAssemblyLoaderThatEnsuresFileBeingWatched : IAnalyzerAssemblyLoader
```

Comment:
>private class AnalyzerAssemblyLoaderThatEnsuresFileBeingWatched : IAnalyzerAssemblyLoad [](start = 8, length = 87)

Removing these classes for now since they don't currently do anything useful and we'll need to set up file watcher differently, since we are no longer loading analyzers in devenv process.